lectures/calvo_machine_learn.md

```diff
@@ -567,7 +567,7 @@ def compute_θ(μ, α=1):
     
     # Compute the weighted sums for all t
     weighted_sums = jnp.array(
-        [jnp.sum(λ_powers[:T-t] * μ[t:T]) for t in range(T)])
```

Comment:
@HumphreyYang should we do this for `jnp.sum` as well as `np.sum`?

Reply:
Many thanks @mmcky, I think it's fine because they behave similarly.

I will do another pass over every lecture once the preview PR is opened!